### PR TITLE
patch values stacked plot

### PR DESCRIPTION
**Pull request by juanitorduz**
Closes https://github.com/pymc-labs/pymc-marketing/issues/1148

Simply clip the contributions and add a warning.

----
📚 Documentation preview 📚: https://pymc-marketing--1181.org.readthedocs.build/en/1181/

### CODE REVIEW

File: pymc_marketing/mmm/base.py [hunk 1]
```diff
@@ -1126,7 +1127,14 @@ def plot_grouped_contribution_breakdown_over_time(
...
             area_params.update(area_kwargs)
-        all_contributions_over_time.plot.area(**area_params)
+        try:
+            all_contributions_over_time.plot.area(**area_params)
+        except ValueError:
+            warnings.warn(
+                "Each contribution value must be either all positive or all negative",
+                stacklevel=2,
+            )
+            return None
```
Comment (juanitorduz) [inline pymc_marketing/mmm/base.py:1137]: I do not think there is an easy fix. It is a limitation of the plot. 

Any other ideas?

Comment (wd60622) [reply pymc_marketing/mmm/base.py:1137]: This is what comes to my mind. We were basically just lucky that posterior estimates where positive for "Base" group.

Comment (wd60622) [reply pymc_marketing/mmm/base.py:1137]: I would just return the figure still

File: pymc_marketing/mmm/base.py [hunk 2]
```diff
@@ -1126,7 +1127,17 @@ def plot_grouped_contribution_breakdown_over_time(
         area_params = dict(stacked=True, ax=ax)
         if area_kwargs is not None:
             area_params.update(area_kwargs)
-        all_contributions_over_time.plot.area(**area_params)
+        try:
+            all_contributions_over_time.plot.area(**area_params)
```
Comment (wd60622) [inline pymc_marketing/mmm/base.py:1131]: Thoughts of check on frame values instead of try except?

**Review by wd60622 — APPROVED** [review]
Not blocker. Lets get fix through

File: pymc_marketing/mmm/base.py (hunk 2 above)
Comment (juanitorduz) [reply pymc_marketing/mmm/base.py:1131]: The cheks are done in the plot.area function itself and can have many edge cases. For now lets direct the responsibility to matplotlib 🙏

**Pull request merged**